imgui.h

```diff
@@ -897,7 +898,9 @@ enum ImGuiDragDropFlags_
     ImGuiDragDropFlags_AcceptBeforeDelivery         = 1 << 10,  // AcceptDragDropPayload() will returns true even before the mouse button is released. You can then call IsDelivery() to test if the payload needs to be delivered.
     ImGuiDragDropFlags_AcceptNoDrawDefaultRect      = 1 << 11,  // Do not draw the default highlight rectangle when hovering over target.
     ImGuiDragDropFlags_AcceptNoPreviewTooltip       = 1 << 12,  // Request hiding the BeginDragDropSource tooltip from the BeginDragDropTarget site.
-    ImGuiDragDropFlags_AcceptPeekOnly               = ImGuiDragDropFlags_AcceptBeforeDelivery | ImGuiDragDropFlags_AcceptNoDrawDefaultRect  // For peeking ahead and inspecting the payload before delivery.
+    ImGuiDragDropFlags_AcceptPeekOnly               = ImGuiDragDropFlags_AcceptBeforeDelivery | ImGuiDragDropFlags_AcceptNoDrawDefaultRect,  // For peeking ahead and inspecting the payload before delivery.
+    ImGuiDragDropFlags_AcceptNoBorderPadding        = 1 << 13,  // Disable extra padding of border rendered around drop destination.
```

Comment:
Inconsistent naming between `_AcceptNoBorderPadding` vs `style.AcceptDropPadding` ?

I think `Border` could be renamed, is a reference to how we draw this drop target but we could imagine it would became a solid thin rectangle, so we should focus on semantic e.g. `_AcceptNoPadding`

And `style.DragDropAcceptPadding`.
But I'm not sure we need this style variable at all for now unless you have a clear use for it.